content/get-haiku/release-notes-r1b2.md

```diff
@@ -0,0 +1,271 @@
+ - [ sys/resource.h](https://git.haiku-os.org/haiku/commit/?id=8ae2e95643ee7ba3f46b0e8a73df1466f23aa7be) Add `[gs]etpriority()` from the POSIX-1.2013 standard.
+ - [unistd.h](https://git.haiku-os.org/haiku/commit/?h=hrev53464&id=af0281a8c20afe53d805b5f3f14935a59b214bdb) Add the `_SC_HOST_NAME_MAX`, `_SC_REGEXP`, `_SC_SYMLOOP_MAX`, and `_SC_SHELL` for `sysconf()`
+ - [unistd.h](https://git.haiku-os.org/haiku/commit/?h=hrev52776&id=b9c25b0d0ec7bb16d72dca4b6f529af604e66df6) Add `nice()` from the POSIX.1-2008 standard.
+
+### Game Kit
+ - [FileGameSound.h](https://git.haiku-os.org/haiku/commit/?h=hrev53625&id=69f814cded60c5f977a61a2ab747e4456323e6f2) Allow initialisation of the `BFileGameSound` from a `DataIO` object, to allow for opening game sound files from sources other than files.
+
+### Interface Kit
+ - [Deskbar.h](https://git.haiku-os.org/haiku/commit/?h=hrev52499&id=efafab643ce980e3f3c916795ed302599f6b4f66) Add MaxItemWidth() and MaxItemHeight() as part of the work to support variable size icons for Deskbar replicants.
+ - [View.h](https://git.haiku-os.org/haiku/commit/headers/os/interface/View.h?id=5bbf7f1be073d376e37da34681c8e6c7acfcf198) New `B_MOUSE_BUTTON` macro that helps you identify which mouse button was clicked on a mouse with more than three buttons.
+ - [View.h](https://git.haiku-os.org/haiku/commit/headers/os/interface/View.h?id=47102c074278637c3b33935b451d53f7ad176658) Add a the new `B_TRANSPARENT_BACKGROUND`, to identify views that have a transparant background (i.e. the background will not be drawn for those views).
+ - [View.h](https://git.haiku-os.org/haiku/commit/headers/os/interface/View.h?id=c67dde0f2ccb45d550af410cf6d73fee39d10504) Add Tiled Background methods to `BView` objects to asynchronously draw bitmaps as background tiles for the view, using the overloaded `DrawTiledBitmapAsync()` method.
+
+### Locale Kit
+ - [Country.h](https://git.haiku-os.org/haiku/commit/headers/os/locale/Country.h?id=70cdd7d4f5fc62e8b3e220646f84235ec3d444d5) Add a `SetTo()` and an `InitCheck()` method to the `BCountry` class.
+
+### Package Kit
+ - [PackageRoster.h](https://git.haiku-os.org/haiku/commit/headers/os/package/PackageRoster.h?id=e2c7bb900cf83fe0a3834ad2189577687ffaa03f) Add a `IsRebootNeeded()` method to the `BPackageRoster` class.
+ - [PackageRoster.h](https://git.haiku-os.org/haiku/commit/headers/os/package/PackageRoster.h?id=b711002d345e2e9e0075be46a37492624f73ccdd) Add a `IsPackageActive()` method to the `BPackageRoster` class.
+
+## New contributor
+
+Since the last release, Kyle Ambroff-Kao received commit access.
+
+## Source code
+
+The source code of Haiku itself, the source code of the required build tools and the optional packages (except for closed source ones) is contained within the release images as the `_source` packages (except on the "CD" image, which it was left out of due to space constraints.)
```

Comment:
There is no "CD" image. We release a single DVD/anyboot image that includes the sourcecode.

For the closed source packages I would prefer that we clearly list what this includes, because it's a common FAQ that we're not "completely open source" or something thanks to a mention about that on the FSF website. As far as I know, it's only wifi firmwares, and liblayout which is needed only by Wonderbrush in the release image.